src/apps/backend/modules/booking-availability/types.ts

```diff
@@ -0,0 +1,17 @@
+import { ToolCall } from '../common-types/common';
+
+export interface AvailableBookingSlotParams {
+  showForDays?: number;
+  startDate?: Date;
+}
+
+export interface BookingAvailabilityCustomRequestBody {
+  message: {
+    toolCalls: ToolCall<AvailableBookingSlotParams>[];
+  };
+}
+
+export interface AvailableBookingSlot {
+  end_time: string;
```

Comment:
This should be `endTime`, `startTime` and their type should be `Date`.

When the vapi controller serializes this type, it should be converted at that layer. Service should return correct data typed return value.